src/game/items/CItem.cpp

```diff
@@ -49,6 +49,7 @@ lpctstr const CItem::sm_szTrigName[ITRIG_QTY+1] =	// static
 	"@Buy",
 	"@Click",
 	"@ClientTooltip",	// Sending tooltip to a client
+	"@ClientTooltip_Sent",
```

Comment:
Out of curiosity, what are the use cases of this new trigger?

Reply:
The reason to add new trigger is before remove or replace the cliloc, it should be send by sphere so we need another trigger which trigger after default sphere clilocs to use REPLACECLILOC and REMOVECLILOC. Because @ClientTooltip is trigger before default sphere tooltips send so when you use replacecliloc it doesn't see default clilocs because it doesn't send yet. I didn't want to move @ClientTooltip under default cliloc because RETURN 1 not working anymore to block default cliloc, so I add another trigger which triggered after default cliloc sent.

Reply:
Got it! What about renaming ClientTooltip_Sent to something more self-explanatory like ClientTooltipDefault? ClientTooltip_Sent makes me think that it triggers after the tooltip list is sent to the client, not that we are after default tooltips are generated but before they are sent

Reply:
Basically where I put the trigger is meaning nearly sent, I mean;

The query of the process is;
- Start function
- Call @ClientTooltip trigger
- Sent default aos cliloc list
- Call @ClientTooltip_Sent trigger
- Finish function

This is why I chose add _SENT, we can't use @ClientTooltipDefault, because when you use RETURN 1 under @ClientTooltip, default tooltips doesn't send but @ClientTooltip_Sent still trigger to replace or remove clilocs which set under @ClientTooltip. We can maybe use @ClientTooltip_AfterDefault or something like this but it is longer and hard to remember, but we can use anything specific to make sure it's understandable

Reply:
Checking the code, to me it looks that the following is happening, correct me if i'm wrong:
- Start function
- Call @ClientTooltip trigger 
- If there's no RETURN 1, add to the tooltip cache the default tooltips
- Call @ClientTooltip_Sent

Reply:
Yes, I forgot to specify for hash calculate, yes this is the place where ClientTooltip_Sent triggered.

Reply:
Maybe we can change the trigger as @ClilocSent or @ClientTooltip_Cliloc(Sent)

Reply:
Still i consider a "sent tooltip" a tooltip for which the packets have been sent to the client, but that's actually happening after ClientTooltip_Sent, the only thing happening between ClientTooltip and ClientTooltip_Sent is the default tooltips being added to the list, so i can't really understand why you are using the "sent" word to name that tooltip, to me it makes more sense to give a name that recalls to the fact that the default tooltips were just generated

Reply:
Honestly I haven't thing about it that much, I just want to make similar with ClientTooltip, and I thought clilocs already sent before that trigger so I just make is @ClientTooltip_Sent. But I can change it to @ClientTooltip_AfterCliloc or @ClientTooltip_ClilocSent to make more understandable. I haven't think about tooltip is a packet, I just think it's a trigger which triggers after clilocs sent, so I just add _Sent at the end.

Reply:
I replace the trigger with @ClientTooltip_AfterDefault to be more specific so people can understand easily that can use add/replace/removecliloc under @ClientTooltip_AfterDefault which trigger after default clilocs sent by server.